frontend/src/core/comments/index.js

```diff
@@ -0,0 +1,5 @@
+/* @flow */
+
+export { default as CommentsList } from './components/CommentsList';
+export { default as Comment } from './components/Comment';
+export { default as AddComment } from './components/AddComment';
```

Comment:
Nit: these three items should be in reverse order (sorted alphabetically by their module name).